ruby-challenges.rb

```diff
@@ -10,23 +10,91 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+# PSEUDOCODE
+# create method with_letter
+# input: array of words and single letter
+# output: array of all words containing that letter
+
```

Comment:
include the process of how you will access the data and how you will manipulate it